packages/kbn-generate-csv/types.ts

```diff
@@ -11,8 +11,10 @@ import type { SerializedSearchSourceFields } from '@kbn/data-plugin/public';
 /**
  * Duplicated from @kbn/reporting-export-types-csv-common to reduce dependencies
  */
+export type CsvPagingStrategy = 'pit' | 'scroll';
 export interface JobParamsCSV {
   browserTimezone?: string;
   searchSource: SerializedSearchSourceFields;
   columns?: string[];
+  searchStrategy?: CsvPagingStrategy;
```

Comment:
let's call it `pagingStrategy`?  I almost confused this with search strategies from data service